docs/experiments-msmarco-irst.md

```diff
@@ -0,0 +1,226 @@
+# Pyserini: IRST on MS MARCO Passage and Document
+
+This page describes how to reproduce IRST experiments with the IBM model on the MS MARCO collections.
+
+
+## Passage Reranking 
+
+### Data Preprocessing
+
+For IRST, we make the corpus as well as the pre-built indexes available to download.
+
+> You can skip the data prep and indexing steps if you use our pre-built indexes. Skip directly down to next section below.
+
+Here, we start from MS MARCO [passage corpus](https://github.com/castorini/pyserini/blob/master/docs/experiments-ltr-msmarco-passage-reranking.md) that has already been processed.
+As an alternative, we also make available pre-built indexes (in which case the indexing step can be skipped).
+
+
+We run the below commands to obtain pre-built index in cache.
+
+```bash
+python -c "from pyserini.search.lucene import LuceneSearcher; LuceneSearcher.from_prebuilt_index('msmarco-passage-ltr')"
```

Comment:
You shouldn't need to do this explicitly... if you specify command below, pyserini will download automatically...